Homework8_1/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using static Homework8_1.Array;
```

Comment:
Why do you need this using? You can call static method via class name

Reply:
fixed.